sale_order_lot_selection/tests/test_stock_lot_add_to_sale_order.py

```
@@ -0,0 +1,132 @@
+                            "lot_id": self.lot_cable.id,
+                            "quantity": self.lot_cable.product_qty,
+                        },
+                    )
+                ],
+            }
+        )
+
+        wizard.action_add_lots_to_sale_order()
+
+        sale_order = self.sale_empty
+        lines = sale_order.order_line.filtered(lambda l: l.lot_id == self.lot_cable)
```

Ruff (0.8.2)

Ambiguous variable name: `l`

(E741)